docs/source/example_input/boosted_frame_script.py

```diff
@@ -40,13 +40,18 @@
 Nr = 75          # Number of gridpoints along r
 rmax = 150.e-6   # Length of the box along r (meters)
 Nm = 2           # Number of modes used
-n_guard = 40     # Number of guard cells
-exchange_period = 10
+
 # The simulation timestep
 dt = (zmax-zmin)/Nz/c   # Timestep (seconds)
 N_step = 101     # Number of iterations to perform
                  # (increase this number for a real simulation)
 
+# Stencil order of the solver. -1 corresponds to the standard
+# fully spectral solver with infinite order. For a localized stencil,
+# thus finite-order accuracy of the solver, choose n_order > 4
+# (needs to be a multiple of 2).
```

Comment:
Note: the above text should probably be rectified so as to fit within an 80 character line.